### PR TITLE
Section LP language

### PR DESCRIPTION
**Pull request by RacheltheEditor**
Updates to introductory lines for clarity of section content in the context of section LPs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1528,9 +1525,8 @@ the **settlement date**.
 
 ### Less common types of equity
 
-We’ll finish our tour of the ways equity can be granted with some other, less common kinds
-of equity. Most employees won’t run into these, but in the interest of completeness, it’s
-worth mentioning a few other flavors of equity compensation out there.
+Most employees won’t run into these, but a complete tour of the ways equity can be granted
```
Comment (jlevy) [inline README.md]: Change to:
While most employee equity compensation takes the form of stock, stock options, or RSUs, a complete tour of equity compensation must mention a few less common forms.

(Nit is imo equity can have a flavor but "ways" themselves do not." 😂 )

File: README.md [hunk 2]
```diff
@@ -1880,16 +1876,17 @@ Any good resources on this?
 
 ## Taxes on Equity Compensation
 
-Now that we’ve covered the basic concepts of equity and taxes, we’ll get into some messy
-details of how they interact.
-
-As already discussed, employees can get restricted stock, stock options, or RSUs.
-The tax consequences for each of these is dramatically different.
+It is crucial to understand the tax implications of stock options and awards.
```
Comment (jlevy) [inline README.md]: This changes sense. The first sentence now neglects RSUs.

I'll take another pass at rewriting these.

File: README.md [hunk 3]
```diff
@@ -1880,16 +1876,17 @@ Any good resources on this?
...
+Equity and taxes interact in complicated ways, and the tax consequences for an employee’s
```
Comment (jlevy) [inline README.md]: Language issue: It's tax consequences not "for" your equity but "of receiving" equity.

File: README.md [hunk 4]
```diff
@@ -1880,16 +1876,17 @@ Any good resources on this?
...
+restricted stock, stock options, or RSUs are dramatically different.
+This section will cover these messy details and help you avoid outsized taxes on your
```
Comment (jlevy) [inline README.md]: "avoid outsized ..." -> "plan and make decisions that reduce the tax burden of your equity compensation"

Comment (jlevy) [reply README.md]: (often you can't avoid taxes, but you can plan for them)

File: README.md (hunk 1 above)
Comment (RacheltheEditor) [reply README.md]: ...the ways are the flavors? I did not like that expression anyway so thank you for fixing 😂

File: README.md (hunk 2 above)
Comment (RacheltheEditor) [reply README.md]: I just cut the first sentence, it was padding.

File: README.md (hunk 3 above)
Comment (RacheltheEditor) [reply README.md]: good catch

Comment (RacheltheEditor) [reply README.md]: fixed ("for an employee receiving")

Comment (RacheltheEditor) [reply README.md]: The "for" is better in that context. The consequences are consequences for the employee. Without the employee, they don't exist. The "tax consequences of an employee receiving" could be consequences for anyone or anything. The old version made that mistake ("The tax consequences for each of these" should have been "of)

File: README.md (hunk 4 above)
Comment (RacheltheEditor) [reply README.md]: fixed

Comment (RacheltheEditor) [reply README.md]: (cut "plan and")

**Pull request merged**